docs/getting-started/cli-console-app-tutorial.md

```diff
@@ -1,8 +1,8 @@
-# Writing Console Apps: A Step by Step Guide
+# Using the CLI tools to write console apps: A step-by-step guide
```

Comment:
Maybe the title could be:
Writing .NET Core console apps using the CLI tools: A step-by-step guide

I think it can help with SEO to have the .NET Core keyword in there.